src/accumulator.jl

```diff
@@ -235,3 +236,18 @@ function Base.intersect!(a::Accumulator, b::Accumulator)
     end
     return a
 end
+function Base.show(io::IO, acc::Accumulator{T,V}) where {T,V}
+    l = length(acc)
+    if l>0
+        print(io, "Accumulator(")
+    else
+        print(io,"Accumulator{$T,$V}(")
+    end
+    for (count, (k, v)) in enumerate(acc)
+        print(io, k, " => ", v)
+        if count < l
+            print(io, ", ")
+        end
+    end
+    print(io, ")")
+end
```

Comment:
Is this change needed?

Reply:
Yes I think , since a very important point  is mentioned in the issue regarding the space after pairs are printed i.e.
 `(3 => 2,2  => 1)` printed as `(3 => 2, 2 => 1)`.
Also I think it is better to have Overridden Base.show method for it might be useful in future operations.

Reply:
Fair, thanks

Reply:
Fair, thanks